templates/_layouts/docs.html

```diff
@@ -112,6 +112,7 @@
                 {{ side_nav_item("/docs/utilities/floats", "Floats") }}
                 {{ side_nav_item("/docs/utilities/font-metrics", "Font metrics") }}
                 {{ side_nav_item("/docs/utilities/functions", "Functions") }}
+                {{ side_nav_item("/docs/utilities/has-icon", "Has-icon", "New", "information") }}
```

Comment:
It would be good to make title more human-friendly.

```suggestion
                {{ side_nav_item("/docs/utilities/has-icon", "Text with icon", "New", "information") }}
```